### PR TITLE
Code Review

### PR DESCRIPTION
**Pull request by AntonioHincapie**
- Use React documentation.
- Use React components.
- Use React props.
- Use React Router.
- Connect React and Redux.
- Handle events in a React app.
- Write integration tests with a React testing library.
- Use styles in a React app.
- Use React life cycle methods.
- Apply React best practices and language style guides in code.
- Use store, actions, and reducers in React.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,72 @@
...
-### Deployment
+Give a ⭐️ if you like this project!
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+## Acknowledgments
 
-### `npm run build` fails to minify
+- Microverse curriculum.
+- My code partners.
+- My code reviewers.
```
Comment (alexander16108) [inline README.md]: this README should contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) 👍🏾

File: src/Components/Details/Details.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+          <img src={closeArrow} alt="Back" />
+        </NavLink>
+        <div
+          className="containerCrypto"
+          id={id}
+        >
+          <img src={Crypto.img} alt={Crypto.name} />
+          <h3>
+            Rank #
+            {Crypto.rank}
```
Comment (alexander16108) [inline src/Components/Details/Details.js:25]: this component along with the other component should have a unit test for the app and some pure functions in Redux code. 💯

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @AntonioHincapie , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- passing linters ✔️ 
- descriptive readme ✔️ 
- good pr ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @AntonioHincapie, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**